packages/api/src/import_object.rs

```diff
@@ -186,6 +200,43 @@ impl ImportObject {
         self.inner.contains_namespace(namespace_name)
     }
 
+    /// Gets an `ImportObject` from a Python dictionary.
```

Comment:
This comment seems to be the wrong way around.

Reply:
Ooops